### DISCUSSION TITLE
Why is the Mastodon iOS application asking a user before they unfollow another user while they have specifically indicated that they do not wish to be asked before they unfollow another user?

### DISCUSSION THREAD

**Discussion by Diwannist** [mastodon/mastodon#32783] [Q&A] [answered]
![IMG_7388](https://github.com/user-attachments/assets/41cbfe08-ee2e-40f8-8b9a-a8facf7503df)
![IMG_7389](https://github.com/user-attachments/assets/88d21851-b3e7-4930-ad5a-211894123022)

**Comment by Gargron** [comment, marked as answer]
It sounds like a bug. You can open a bug report on https://github.com/mastodon/mastodon-ios
